docs/release/trg-2/trg-2-4.md

```diff
@@ -2,31 +2,34 @@
 title: TRG 2.04 - Leading product repository
 ---
 
-| Author                | Status | Created     | Post-History                               |
-|-----------------------|--------|-------------|--------------------------------------------|
-| Catena-X System Team  |        | 10-Jan-2023 | Initial release                            |
+| Status | Created     | Post-History    |
+|--------|-------------|-----------------|
+| Active | 07-Mar-2023 |                 |
+|        | 10-Jan-2023 | Initial release |
+
+## Why
+
+We have multiply steak holders for a repository: System team, release management, test management, external people who are trying it out, want to contribute or need to set it up.
+They need to be able to understand complicated product structures in an intuitive way to have an easier task of diving into a product.
 
 ## Description
 
-Each product should have a leading product repository that can point the users to all the parts/components/charts/releases of that product. This can be achieved by having a single repositry for a product alltogether or creating references to the other repositories.
+Each product **must** have a leading product repository that can point the users to all the parts/components/charts/releases of that product.
+This can be achieved by having a single repository for a product or creating references to the other repositories.
 
-A good example when a product has a separate repository for frontend and backend. Then the product team should create a leading repository where they reference the components/repositories of the product:
+A good example when a product has a separate repository for frontend and backend. Then the product team **must** create a leading repository where they reference the components/repositories of the product:
```

Comment:
must create -> or declare? Can we give options here?

Reply:
I guess it's up to the team. If they want to declare an already existing repo as leading we should allow that.

Reply:
i worded it better